### PR TITLE
feat: uninstall command `nrm`

### LINKED ISSUE #15: `npm uninstall` / `yarn remove`
Hey, what about `npm uninstall` / `yarn remove` command? Is it available?

### PR DESCRIPTION
**Pull request by vltansky**
closes #15

### CODE REVIEW

File: src/commands.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ export function getCommand(
 
   if (!c)
     throw new Error(`Command "${command}" is not support by agent "${agent}"`)
-
+// console.log(c.replace('{0}', args.join(' ')).trim());
```
Comment (antfu) [inline src/commands.ts:19]:
```suggestion

```

File: src/nu.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { parseNu } from './commands'
+import { parseNrm } from './commands'
```
Comment (antfu) [inline src/nu.ts:1]: Accident?

File: package.json [hunk 3]
```diff
@@ -34,6 +35,7 @@
     "nr": "esno src/nr.ts",
     "nu": "esno src/nu.ts",
     "nx": "esno src/nx.ts",
+    "nrm": "esno src/nrm.ts",
     "dev": "esno src/ni.ts",
     "build": "rimraf dist && tsup src/ni.ts src/nci.ts src/nr.ts src/nu.ts src/nx.ts src/index.ts --format cjs,esm --dts",
```
Comment (antfu) [inline package.json:40]: Need to add build target as well

Comment (vltansky) [reply package.json]: done

File: src/nu.ts (hunk 2 above)
Comment (vltansky) [reply src/nu.ts]: yep, sorry

**Pull request merged**